njsscan/rules/pattern_matcher/template_rules.yaml

```diff
@@ -69,13 +69,25 @@
   message: The Squirrelly.js template has an unescaped variable. Untrusted user input
     passed to this variable results in Cross Site Scripting (XSS)
   type: Regex
-  pattern: '{{.+\|.*safe.*}}'
+  pattern: '{{(?!.*(?:safeURL|safeHTML|safeCSS|safeJS|safeJSStr|safeHTMLAttr)).*\|.*safe.*}}'
```

Comment:
There is a problem with this regex, it will skip variable names with safeURL/.. in them.

https://regex101.com/r/a3QdmS/1

Reply:
I have excluded all Hugo-related keywords from this rule and used those keywords in the specific Hugo rule. Otherwise I'll still have error on hugo SAFE keyword instead of the warning.

Reply:
I think you need to move the Hugo specific keywords after `.*\|` so that it ignores something like `{{ foo | safeURL }}` but still catch`{{ safeURL | safe }}` for this Squirrelly.js rule.